commons/common2.ml

```diff
@@ -1440,6 +1440,17 @@ let int_of_all s =
   if String.length s >= 2 && (String.get s 0 =<= '0') && is_digit (String.get s 1)
   then int_of_octal s else int_of_string s
 
+let int_of_string_c_octal_opt s =
```

Comment:
There is a similar function in semgrep in Parse_tree_sitter_helpers.ml

Reply:
Looks like the same code. Was it on purpose? Then we should remove it in Parse_tree_sitter_helpers.ml no?

Reply:
That's right, I copied it from there, and I will remove it from `Parse_tree_sitter_helpers` when we update Pfff in Semgrep.